Timeframe for indicator Timeline query updated (#3034)
elastic/security-docs: docs/cases/indicators-of-compromise.asciidoc

[[find-related-sec-events]]
== Find related security events

Investigate an indicator in <<timelines-ui,Timeline>> to find related security events in your environment. You can add an indicator to Timeline from the Indicators table or the Indicator details flyout.

[role="screenshot"]
image::images/indicator-query-timeline.png[Shows the results of an indicator being investigated in Timeline]

> When you add an indicator to Timeline, a new Timeline opens with an auto-generated KQL query. The query includes the selected indicator and related fields mapped from relevant source events. The query's time range is automatically set to seven days before and seven after the indicator's `timestamp`. 

Author: ```suggestion
When you add an indicator to Timeline, a new Timeline opens with an auto-generated KQL query. The query contains the indicator field-value pair that you selected plus the field-value pair of the automactially mapped source event. By default, the query's time range is set to seven days before and seven after the indicator's `timestamp`. 
```

> When you add an indicator to Timeline, a new Timeline opens with an auto-generated KQL query. The query contains the indicator field-value pair that you selected plus the field-value pair of the automactially mapped source event. By default, the query's time range is set to seven days before and seven after the indicator's `timestamp`. 

Reviewer: ```suggestion
When you add an indicator to Timeline, a new Timeline opens with an auto-generated KQL query. The query contains the indicator field-value pair that you selected plus the field-value pair of the automactially mapped source event. By default, the query's time range is set to seven days before and after the indicator's `timestamp`. 
```

> You can use the Timeline query results to identify and predict related events. They can also help you assess and respond to potential threats.

Reviewer: Maybe move this to the first paragraph, above the image? Might be nice to get a sense of why the Timeline query results would be helpful/useful, before going into nuts & bolts of how it functions.

[discrete]
[[example-indicator-timeline]]
=== Example indicator Timeline investigation  

> The following image shows a file hash indictor being investigated in Timeline. The indicator field and value pair is:

Reviewer: To match format used above:
```suggestion
The following image shows a file hash indictor being investigated in Timeline. The indicator field-value pair is:
```

`threat.indicator.file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`

[role="screenshot"]
image::images/indicator-in-timeline.png[Shows the results of an indicator being investigated in Timeline]

> The auto-generated query contains the indicator and the related `file.hash.sha256` field:

Author: ```suggestion
The auto-generated query contains the indicator field-value pair (mentioned previously) and the auto-mapped source event field-value pair, which is:
```

`file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`

> The query results show a related alert that has a matching `file.hash.sha256` field value. This alert can be further analyzed to determine if it suggests suspicious or malicious activity in the environment.  

Reviewer 2: ```suggestion
The query results show an alert with a matching `file.hash.sha256` field value, which may indicate suspicious or malicious activity in the environment.  
```

> The query results sh/Users/nastashasolomon/Documents/Elastic/security-docs/docs/cases/images/indicator-in-timeline.pngow a related alert that has a matching `file.hash.sha256` field value. This alert can be further analyzed to determine if it suggests suspicious or malicious activity in the environment.  

Author: It is, I'll fix this right now. Thanks!

> The query results sh/Users/nastashasolomon/Documents/Elastic/security-docs/docs/cases/images/indicator-in-timeline.pngow a related alert that has a matching `file.hash.sha256` field value. This alert can be further analyzed to determine if it suggests suspicious or malicious activity in the environment.  

Author: ```suggestion
The query results show a related alert that has a matching `file.hash.sha256` field value. This alert can be further analyzed to determine if it suggests suspicious or malicious activity in the environment.  
```
